packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/quanticGeneratedAnswerContent.js

```diff
@@ -11,6 +11,7 @@ import generatedTextContentTemplate from './templates/generatedTextContent.html'
 /**
  * The `QuanticGeneratedAnswerContent` component displays the generated answer content.
  * @category Internal
+ * @fires CustomEvent#quantic__answergenerated
```

Comment:
changes in this file have to do with : https://coveord.atlassian.net/browse/SFINT-5914